src/test/java/org/owasp/esapi/reference/ValidatorTest.java

```diff
@@ -297,7 +297,7 @@ public void testGetValidSafeHTML() throws Exception {
 
         assertEquals("Test. &lt;<div>load=alert()</div>", rule.getSafe("test", "Test. <<div on<script></script>load=alert()"));
         assertEquals("Test. <div>b</div>", rule.getSafe("test", "Test. <div style={xss:expression(xss)}>b</div>"));
-        assertEquals("Test.", rule.getSafe("test", "Test. <s%00cript>alert(document.cookie)</script>"));
+        assertEquals("Test.", rule.getSafe("test", "Test. <script>alert(document.cookie)</script>"));

```

Comment:
Hello,
Sorry for this absurdity. 
Is it normal that the assertEquals expected object does not correspond to the actual safe object return by the HTMLValidationRule class?

I think the expected object should be equal to "Test. alert(document.cookie)". What is your opinion? maybe I do not dig where I should.